src/Core/Search.php

```diff
@@ -129,8 +131,8 @@ public static function getContactsFromGlobalDirectory(string $search, int $type
 
 		$resultList = new ResultList(
 			($results['page']         ?? 0) ?: 1,
-			 $results['count']        ?? 0,
-			($results['itemsperpage'] ?? 0) ?: 30
+			($results['itemsperpage'] ?? 0) ?: 30,
+			$results['count']        ?? 0
```

Comment:
Why did you swap these? The parameter order is `page, count, itemsperpage`.
```suggestion
			$results['count']         ?? 0,
			($results['itemsperpage'] ?? 0) ?: 30
```

Reply:
You re right.